tests/test_definitions/test_expectations_cfe.py

```diff
@@ -50,10 +50,26 @@ def pytest_generate_tests(metafunc):
 
                 for d in test_configuration["datasets"]:
                     datasets = []
+                    # optional only_for and suppress_test flag at the datasets-level that can prevent data being
+                    # added to incompatible backends. Currently only used by expect_column_values_to_be_unique
+                    only_for = d.get("only_for", None)
+                    if only_for and not isinstance(only_for, list):
+                        raise ValueError(
+                            "Invalid test specification. 'only_for' should be passed in as a list"
+                        )
+                    suppress_test_for = d.get("suppress_test_for", None)
+                    if suppress_test_for and not isinstance(suppress_test_for, list):
+                        raise ValueError(
+                            "Invalid test specification. 'suppress_test_for' should be passed in as a list"
+                        )
```

Comment:
`None` the default value for `get`'s second arg so it's not entirely necessary here - but that's being very nitpicky and I don't mind the explicitness.

Will defer to you here!

Reply:
Whatever you pick, please be consistent!

Reply:
Would it be wrong to coerce these vars to be lists? (i.e. `only_for = [only_for]`)?

Reply:
@kenwade4  I like it!